src/main.ts

```diff
@@ -1,31 +1,30 @@
 import * as core from '@actions/core'
 import {Inputs, createPullRequest} from './create-pull-request'
 import {inspect} from 'util'
-import * as utils from './utils'
 
 async function run(): Promise<void> {
   try {
     const inputs: Inputs = {
       token: core.getInput('token'),
       path: core.getInput('path'),
-      addPaths: utils.getInputAsArray('add-paths'),
+      addPaths: core.getMultilineInput('add-paths'),
       commitMessage: core.getInput('commit-message'),
       committer: core.getInput('committer'),
       author: core.getInput('author'),
-      signoff: core.getInput('signoff') === 'true',
+      signoff: core.getBooleanInput('signoff'),
```

Comment:
I'm happy to switch to `getBooleanInput` because there is no risk of a breaking change for users here. 👍